packages/expo-cli/jest/unit-test-config.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+
+module.exports = {
+  rootDir: path.resolve(__dirname, '..'),
+  roots: ['__mocks__', 'src'],
+  displayName: require('../package.json').name,
+  testRegex: '/__(tests|testfixtures)__/.*(test|spec)\\.(j|t)sx?$',
```

Comment:
The default setting is `(/__tests__/.*|(\\.|/)(test|spec))\\.[jt]sx?$`, which should work. `__testfixtures__` is a special path used by `testUtils` of `jscodeshift` (we use this in `expo-codemod`), but that directory should not include regular tests.
```suggestion
```